Samples/ExtendedControlAndMetadata/EyeGazeAndBackgroundSegmentation/CameraKsPropertyHelper/KSHelper.h

```diff
@@ -0,0 +1,137 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#pragma once
+
+namespace winrt::CameraKsPropertyHelper::implementation
+{
+    
+    // Redefining structs normally available in ksmedia.h but not available due to WINAPI_PARTITION_APP
+    //
+#pragma region KsMediaRedefinition
+    struct KSPROPERTY
+    {
+        byte Set[16];
+        unsigned long Id;
+        unsigned long Flags;
+
+        KSPROPERTY(winrt::guid set, unsigned long id = 0, unsigned long flags = 0)
+        {
+            memcpy(Set, &set, sizeof(Set));
+            Id = id;
+            Flags = flags;
+        }
+        KSPROPERTY() : KSPROPERTY(winrt::guid()) {}
+    };
+
+    struct KSCAMERA_EXTENDEDPROP_HEADER {
+        ULONG               Version;
+        ULONG               PinId;
+        ULONG               Size;
+        ULONG               Result;
+        ULONGLONG           Flags;
+        ULONGLONG           Capability;
+    };
+
+    struct KSCAMERA_EXTENDEDPROP_VALUE {
+        union
+        {
+            double          dbl;
+            ULONGLONG       ull;
+            ULONG           ul;
+            ULARGE_INTEGER  ratio;
+            LONG            l;
+            LONGLONG        ll;
+        };
+    };
+
+    struct KSCAMERA_METADATA_ITEMHEADER
+    {
+        ULONG   MetadataId;
+        ULONG   Size;
+    };
+
+    // BackgroundSegmentation-related
+#pragma region BackgroundSegmentation
+
+    struct KSCAMERA_EXTENDEDPROP_BACKGROUNDSEGMENTATION_CONFIGCAPS
+    {
+        // Output width and height in pixels
+        SIZE Resolution;
+
+        // The maximum frame rate the driver can accommodate  for achieving 
+        // background segmentation for each frame corresponding to Resolution
+        struct
+        {
+            LONG Numerator;
+            LONG Denominator;
+        } MaxFrameRate;
+
+        // The width and height of the mask produced when streaming
+        // with a MediaType corresponding to Resolution and MaxFrameRate
+        SIZE MaskResolution;
+
+        // Optional subtype for which this configuration capability applies. If left 
+        // to zero, all streams conforming the Resolution and MaxFrameRate will support 
+        // background segmentation with the specified MaskResolution.
+        GUID SubType;
+
+    };
+
+    struct KSCAMERA_METADATA_BACKGROUNDSEGMENTATIONMASK
+    {
+        KSCAMERA_METADATA_ITEMHEADER Header;
+        RECT MaskCoverageBoundingBox;
+        SIZE MaskResolution;
+        RECT ForegroundBoundingBox;
+        BYTE MaskData[1]; // Array of mask data of dimension MaskResolution.cx * MaskResolution.cy
+    };
+
+#pragma endregion BackgroundSegmentation
+    //
+    // end of ksmedia.h redefinitions
+    
+
+#pragma endregion KsMediaRedefinition

```

Comment:
nit: spacing

Reply:
I like my #pragmas not indented

Reply:
I just meant the blank lines previously didn't group the pragma with the comment